src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss

```
@@ -9,30 +9,48 @@
     background: $dark;
   }
 
-  .time-section {
+  &__time-section {
     border-right: 1px solid $concrete;
     min-width: 110px;
     margin-right: 10px;
   }
 
-  .event-description {
+  &__event-description {
+    display: block;
     color: $concrete;
     white-space: pre-wrap;
+
+    p {
+      display: inline;
+    }
   }
 
-  .primary {
+  &--primary {
     color: $primaryColor;
   }
 
-  .event-section {
+  &__event-section {
     display: flex;
     width: 100%;
     justify-content: space-between;
   }
 
-  .entry-room-button {
+  &__entry-room-button {
     display: flex;
     align-items: center;
     margin-left: 10px;
   }
+  &__event-time,
+  &__event-name {
+    display: block;
+    font-weight: 700;
+  }
+  &__event-date {
+    display: block;
+    font-weight: 700;
+    color: $concrete;
+  }
+  &__event-host {
+    font-weight: 700;
+  }
```

Please include whitespace between blocks like this.. I know i've asked for this to be followed in the past. It creates extra work and delay for us when I have to keep asking for it on each new PR.

```suggestion
  }

  &__event-time,
  &__event-name {
    display: block;
    font-weight: 700;
  }

  &__event-date {
    display: block;
    font-weight: 700;
    color: $concrete;
  }

  &__event-host {
    font-weight: 700;
  }
```

---

fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/6fc557eeb19867a64d8bbfa9c463c96e3c54531a